src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -617,7 +617,7 @@ int goto_instrument_parse_optionst::doit()
       show_goto_functions(
         goto_model,
         log.get_message_handler(),
-        ui_message_handler.get_ui(),
+        message_handler.get_ui(),
```

Comment:
🍌

Reply:
No need to pass message_handler and ui separately.